compiler/src/main/java/bullet/impl/ComponentProcessingStep.java

```diff
@@ -224,4 +272,22 @@ private boolean isVisibleFrom(Element target, PackageElement from) {
         throw new AssertionError();
     }
   }
+
+  private static int getNextPrime(int value) {
```

Comment:
I saw that you were using that. The documentation makes me wary though. I'd rather use something 100% reliable.